### ISSUE TITLE
Consider moving the instructions for upgrading from pre 1.0 to another doc + point to it.

### ISSUE THREAD

**Issue by stuaxo** [EmilStenstrom/django-components#776]
There's quite a lot of configuration to do to get this working, so simplifying things would be good.

On the docs here - https://github.com/EmilStenstrom/django-components it could be worth seperating the part that talks about older versions, i.e. "If you are upgrading from a pre 1.0 version see this doc" and have a seperate doc, since there is already quite a lot to configure.

Consider that as time goes on it's going to get rarer to need to upgrade from pre 1.0

**Comment by JuroOravec** [comment]
@stuaxo Which part are you referring to? Or generally to all the bits where we explain changes in older versions?

Agree though. Once we hit 1.0.0, then I'd like to remove all mentions of pre-1.0 from the main docs. And maybe point people to the docs of the last pre-1.0 version, which would still contain such notes.

**Comment by JuroOravec** [comment]
Sorry, in the meantime I managed to crash the documentation site 😅 

I finally see what you mean! So you're talking for example about the section "Static files prior to v0.100", correct? Yeah, as I said, some of that should go away once we reach v1.

Moreover, I'm currently splitting up the README to smaller articles (see https://github.com/EmilStenstrom/django-components/pull/775), which should also make things clearer.

---

I think we can keep this issue open as a reminder to remove those pre-v1 notes when getting onto v1.

**Comment by dalito** [comment]
@JuroOravec - Do you want to keep the huge readme? I saw that you kept it in the PR that you reverted (#775 / #781).

A smaller "appetizer"-readme may be enough. I think that having the info at two places (readme + docs) has no advantage. Also on PyPI the TOC links don't work for the huge readme (#576).

A small detail: The gh-pages docs are not yet linked from the right sidebar in GitHub repo landing page (I always look there first).

**Comment by JuroOravec** [comment]
@dalito Nope, I had to temporarily revert it to fix the docs building pipeline. See https://github.com/EmilStenstrom/django-components/pull/780#issuecomment-2501367416.

The intention is to have the documentation live in the Github site, and the README file will contain only the very basics / necessary stuff. See  https://github.com/EmilStenstrom/django-components/pull/775

I found & fixed two bugs today that came up in v0.110. But there's one that I found\*. I might try to fix tomorrow, so I don't want to re-reintrodue the reverted commit just yet.

---

\* The remaining bug is that if you specify multiple JS scripts with a single component using `Component.Media.js`, then the order of scripts is not preserved. So if one script depends on the previous one, such logic will fail, because they will run in parallel.

**Comment by dalito** [comment]
@EmilStenstrom - You could link the new docs also from the right sidebar of the [landing page](https://github.com/EmilStenstrom/django-components/) under "About".
